plugins.json

```diff
@@ -93,17 +93,23 @@
     "name": "Get Environment Variables",
     "package": "netlify-plugin-get-env-vars",
     "repo": "https://github.com/shortdiv/netlify-plugin-get-env-vars",
```

Comment:
Before merging this plugin, what are your thoughts on https://github.com/shortdiv/netlify-plugin-get-env-vars/issues/3?